test/test_data/gcc.py

```diff
@@ -19,13 +19,13 @@
         "package_name": "gcc-10.2.1-3.fc33.x86_64.rpm",
         "product": "gcc",
         "version": "10.2.1",
-        "other_products": [],
+        "other_products": ["docker"],
```

Comment:
This is incorrect.  gcc does not contain docker.  This is an artifact of the bad patterns above and will also need to be fixed (here and elsewhere in the tests)